python/lsst/ts/cbp/mock_server.py

```diff
@@ -204,7 +204,12 @@ async def do_new_azimuth(self, azimuth):
             self.encoders.AZIMUTH.set_position(float(azimuth))
             return ""
         except ValueError:
-            raise  # figure out what the real controller does
+            if float(azimuth) > 45:
```

Comment:
It appears that first you set the position, if that raises then you check it against a few values and truncate it, which assumes two things:
- that set_position will raise ValueError if out of bounds
- that the value will be out of range if and only if set_position raises ValueError

For a reader of the code there is no reason to expect that second assumption to be true, and it requires duplicating the range in two places (inside encoders.AZIMUTH and here).

I strongly suggest you rewrite to fix these issues. One obvious solution is to get the limits from self.encoders.AZIMUTH and apply the truncation **before** calling set_position and eliminate the try/except. If set_position raises ValueError it means you have a bug, which you can catch higher up (along with other unexpected errors).

Similarly for the other axes.

Reply:
Minor request: please rename encoders.AZIMUTH to encoders.azimuth and similarly for the others, to match standard Python conventions. Reserve UPPERCASE for constants (such as enum values).